website/content/docs/concepts/connection-workflows/multi-hop.mdx

```diff
@@ -14,6 +14,16 @@ inbound traffic to route through multiple network enclaves to reach the target s
 Multi-hop sessions allow you to chain together two or more workers
 across multiple networks to form reverse proxy connections between the user and the target, even in complex networks with strict outbound-only policies.
 
+## Remove the requirement for inbound network rules
+
+With a multi-hop deployment, all connections are initiated outbound from the most downstream worker in the chain. When the connection is established between the workers, the proxied connections go over the established connection.
```

Comment:
```suggestion
With a multi-hop deployment, all outbound connections are initiated from the most downstream worker in the chain. After Boundary establishes the initial connection between the workers, it uses the established connection for any subsequent connections.
```

Reply:
Hey @Dan-Heath I would say in the suggestion that the tunnels are established outbound, so `all outbound connections` to me suggests that there maybe inbound connections, but it's always established from the outbound direction, if that makes sense? The rest of the suggestions are great :)

Reply:
That does make sense, thanks @dannyjknights ! I was worried that may have changed the meaning. I edited my suggestion to use your original phrasing there and include my edit to the second sentence. If this sounds good to you, I will approve and get this merged 🎉